src/main/java/redgatesqlci/Utils.java

```diff
@@ -25,13 +25,9 @@ public static boolean runSQLCIWithParams(AbstractBuild build, Launcher launcher,
         String allLocations = "";
         String[] possibleSqlCiLocations = 
             {
-                getEnvironmentVariable("DLMAS_HOME", channel) +  "sqlCI\\sqlci.exe",
-                getEnvironmentVariable("ProgramFiles", channel) + "\\Red Gate\\DLM Automation Suite 1\\sqlCI\\sqlci.exe",
-                getEnvironmentVariable("ProgramFiles", channel) + "\\Red Gate\\SQL Automation Pack 1\\sqlCI\\sqlci.exe",
-                getEnvironmentVariable("ProgramFiles", channel) + "\\Red Gate\\sqlCI\\sqlci.exe",
-                getEnvironmentVariable("ProgramFiles(X86)", channel) + "\\Red Gate\\DLM Automation Suite 1\\sqlCI\\sqlci.exe",
-                getEnvironmentVariable("ProgramFiles(X86)", channel) +  "\\Red Gate\\SQL Automation Pack 1\\sqlCI\\sqlci.exe",
-                getEnvironmentVariable("ProgramFiles(X86)", channel) +  "\\Red Gate\\sqlCI\\sqlci.exe"
+                getEnvironmentVariable("DLMAS_HOME", channel) +  "\\sqlci.ps1",
+                getEnvironmentVariable("ProgramFiles", channel) + "\\Red Gate\\DLM Automation Suite 2\\sqlci.ps1",
```

Comment:
These should be `\\Red Gate\\DLM Automation 2\\sqlci.ps1` (without the word Suite)